dev/build_google_component_image.sh

```diff
@@ -245,7 +246,19 @@ function fix_defaults() {
 
 process_args "$@"
 
-declare -a COMPONENTS=('clouddriver' 'deck' 'echo' 'fiat' 'front50' 'gate' 'igor' 'orca' 'rosco' 'consul' 'vault' 'redis')
+declare -A COMPONENTS=( ['clouddriver']='clouddriver' \
```

Comment:
Can you describe what's going on here in a comment, and point out which names are artifacts and which names are services?